addons/auth/.prettierrc.js

```diff
@@ -4,6 +4,12 @@
  */
 
 module.exports = {
-  singleQuote: true,
-  printWidth: 80,
+  overrides: [
```

Comment:
This probably doesn't need to get changed, we might have committed this previously too